policyengine_us/parameters/gov/states/ne/tax/income/agi/subtractions/subtractions.yaml

```diff
@@ -0,0 +1,16 @@
+description: Nebraska subtracts these sources from adjusted gross income.
+values:
+  2021-01-01:
+    - ne_social_security_subtraction
+  2024-01-01:
+    - ne_social_security_subtraction
+    - taxable_public_pension_income
+    
+
+metadata:
+  unit: list
+  period: year
+  label: Nebraska subtractions from federal adjusted gross income
+  reference:
+    - title: Nebraska Revised Statute | chapter 77 | §2716 
```

Comment:
What sections? I only see 13 and 14 for Social Security

Reply:
20:
![image](https://github.com/user-attachments/assets/2185a696-e4c7-4817-b80b-f8ad513f25cb)

Reply:
Please specify the section in the title. No need for a PR now.